sxw.c

```diff
@@ -1019,23 +1049,23 @@ void _print_debuginfo(void) {
 			//ForEachTranspLayer(t) {
```

Comment:
This line can probably safely be removed.

Reply:
Resolved by 103ca00.